frontend/components/CodeEditor.tsx

```diff
@@ -186,10 +190,12 @@ export function CodeEditor({
               dangerouslySetInnerHTML={{ __html: highlightHTML(hljs, lang, handleNewLines(content)) }}
             ></pre>
             <span
+              ref={refLineNumbers}
               className={
                 "line-number-rows font-mono absolute pointer-events-none text-default-500 top-0 left-1 " +
                 `border-solid border-default-300 border-r-1 ${tst}`
               }
+              style={{ height: `${heightPx}px`, overflow: "hidden" }}
```

Comment:
nitpick: for consistency prefer using `overflow-hidden` classname than using style